typescript/common-resolvers/src/export/format-yolo/export-to-yolo.ts

```diff
@@ -6,6 +6,19 @@ import { getImageName } from "../common";
 
 import { ExportFunction } from "../types";
 
+const groupLabelsByImage = (labelsArray: DbLabel[]) => {
+  return labelsArray
+    ? labelsArray.reduce((acc: { [imageId: string]: DbLabel[] }, label) => {
+        const key = label.imageId;
+        if (!acc[key]) {
+          acc[key] = [];
+        }
+        acc[key].push(label);
+        return acc;
+      }, {})
+    : {};
+};
+
```

Comment:
You can use [lodash groupBy](https://lodash.com/docs/#groupBy) function to do this operation.
I guess you'd end-up with something like:

```suggestion
import { isEmpty, groupBy } from "lodash/fp";
import { ExportFunction } from "../types";

const groupLabelsByImage = (labels: DbLabel[]) =>
  isEmpty(labels) ? {} : groupBy(labels, "imageId");
```